frontend/packages/container-security/src/components/image-manifest-vuln.tsx

```diff
@@ -253,46 +289,68 @@ export const ImageManifestVulnTableRow: RowFunction<ImageManifestVuln> = ({
 
 export const ImageManifestVulnTableHeader = () => [
   {
-    title: 'Image Name',
+    title: i18n.t('container-security~Image Name'),
     sortField: 'spec.image',
     transforms: [sortable],
     props: { className: tableColumnClasses[0] },
   },
   {
-    title: 'Namespace',
+    title: i18n.t('container-security~Namespace'),
     sortField: 'metadata.namespace',
     transforms: [sortable],
     props: { className: tableColumnClasses[1] },
+    id: 'namespace',
   },
   {
-    title: 'Highest Severity',
+    title: i18n.t('container-security~Highest Severity'),
     sortField: 'status.highestSeverity',
     transforms: [sortable],
     props: { className: tableColumnClasses[2] },
   },
   {
-    title: 'Affected Pods',
+    title: i18n.t('container-security~Affected Pods'),
     props: { className: tableColumnClasses[3] },
   },
   {
-    title: 'Fixable',
+    title: i18n.t('container-security~Fixable'),
     sortField: 'status.fixableCount',
     transforms: [sortable],
     props: { className: tableColumnClasses[4] },
   },
   {
-    title: 'Manifest',
+    title: i18n.t('container-security~Total'),
+    sortFunc: 'totalOrder',
+    transforms: [sortable],
     props: { className: tableColumnClasses[5] },
   },
+  {
+    title: i18n.t('container-security~Manifest'),
+    props: { className: tableColumnClasses[6] },
+  },
 ];
 
 export const ImageManifestVulnList: React.FC<ImageManifestVulnListProps> = (props) => {
-  const EmptyMsg = () => <MsgBox title="No Image Vulnerabilities Found" detail="" />;
+  const { t } = useTranslation();
+  const EmptyMsg = () => (
+    <EmptyState variant={EmptyStateVariant.large}>
+      <Title headingLevel="h4" size="lg">
+        <div className="imagemanifestvuln-empty-state__badge-wrapper">
+          <Badge className="imagemanifestvuln-empty-state__badge" isRead>
+            {t('container-security~IMV')}
+          </Badge>
+        </div>
```

Comment:
Shall we extract it and make a generic component, which can be used across can take model or  abbr as prop and i don't think we need to translate this, WDYT?